airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -46,4 +46,4 @@ tests:
       configured_catalog_path: "integration_tests/configured_catalog_report.json"
       ignored_fields:
         "sponsored_products_report_stream": ["updatedAt"]
-      timeout_seconds: 3600
+      timeout_seconds: 3600
```

Comment:
Can you add this "newline" to avoid changes in this file ?

Reply:
resolved